row_047_app.py

```
@@ -0,0 +1,128 @@
+
+app = flask.Flask(__name__)
+model = None
+use_gpu = False
+
+
+def segmentation(image_path):
+    names = re.findall(r'[^\\/:*?"<>|\r\n]+$', image_path)
+    mask_path = output_path + names[0]
+    change_channels_to_rgb(image_path)
+    cmd_predict = "python predict.py -i {0} -o {1}".format(image_path, mask_path)
+    os.system(cmd_predict)
```

Ruff (0.13.2)

Starting a process with a shell, possible injection detected

(S605)